codespell_lib/data/dictionary.txt

```diff
@@ -11427,6 +11431,10 @@ etherenet->Ethernet
 ethnocentricm->ethnocentrism
 ethose->those, ethos,
 etiher->either
+etroneous->erroneous
+etroneously->erroneously
+etror->error
+etrors->errors
```

Comment:
Possibly also suggest terror*?

Reply:
Yes, absolutely. Thanks for the suggestion, added in a76733f24c006e89fc05d4a7a30a4908b113eb1a